src/placeos-core/pulse/heartbeat.cr

```diff
@@ -0,0 +1,46 @@
+require "json"
+
+# require "rest-api"
+
+class Pulse::Heartbeat
```

Comment:
I think this could be better defined wherever the `pulse` service is mounted, or at least define a base type there that can be inherited here.